packages/verification-core/src/verifier/CertifactionClaimVerifier.js

```diff
@@ -305,6 +305,10 @@ export default class CertifactionClaimVerifier {
                 fileEvent.issuer = issuerIdentity
             }
 
+            if (fileEvent.scope === 'retract') {
+                fileEvent.retraction_note = claim.note
```

Comment:
Have you checked the naming of this event attribute with backend? For me it doesn't make sense to call it `retraction_note` because it's just called `note` in the claim and as far as I know, we want to support notes also for the initial signature or registration. And because we only have one event per scope you always know with the scope that it's for example a retraction note.

Reply:
I was not aware of multiple notes when implementing it and wanted to specify it a bit. But you are right, not only retract claims can have notes. So i changed it now just to "fileEvent.note".

Reply:
This needs to be the same in the go verification result, so it would probably make sense to add a subtask to add that also there.